attributes/default.rb

```diff
@@ -1,26 +1,13 @@
-default['openstack-mistral']['install_recipe'] = 'openstack-mistral::install_source'
-
-default['openstack-mistral']['source']['git_url'] = 'https://github.com/stackforge/mistral'
-default['openstack-mistral']['source']['git_revision'] = nil
-default['openstack-mistral']['source']['git_action']   = nil
```

Comment:
Can you also remove from `README.md` obsolete parts?

https://github.com/StackStorm/chef-openstack-mistral#attributes